gammapy/data/observations.py

```diff
@@ -286,29 +308,56 @@ def obs_info(self):
             )
         return meta
 
+    @property
+    def pointing(self):
+        if self._pointing is None:
+            self._pointing = FixedPointingInfo.from_fits_header(self.obs_info)
+        return self._pointing
+
+    def get_pointing_altaz(self, time):
+        """Get the pointing in altaz for given time"""
+        return self.pointing.get_altaz(time, self.observatory_earth_location)
+
+    def get_pointing_icrs(self, time):
+        """Get the pointing in icrs for given time"""
+        return self.pointing.get_icrs(time, self.observatory_earth_location)
+
     @lazyproperty
+    @deprecated(
+        "v1.1",
+        message="Use observation.pointing or observation.get_pointing_{{altaz,icrs}} instead",
+    )
     def fixed_pointing_info(self):
         """Fixed pointing info for this observation (`FixedPointingInfo`)."""
-        return FixedPointingInfo(self.obs_info)
+        return self._pointing
 
     @property
+    @deprecated("v1.1", message="Use observation.get_pointing_icrs(time) instead")
     def pointing_radec(self):
```

Comment:
For convenience one could  still introduce `obs.pointing_icrs_tmid`. Otherwise many people will end up writing `obs.get_pointing_icrs(obs.tmid)` it is not that bad, but the former is still better readable...

Reply:
Actually, my reasoning was to not make it convenient to assume the pointing doesn't change. People should get the "oh right, it might be time-dependent" when trying to get the pointing.